cmd/aergocli/cmd/getpeers.go

```diff
@@ -65,12 +78,74 @@ func execGetPeers(cmd *cobra.Command, args []string) {
 		// TODO show long fields
 		res := jsonrpc.ConvLongPeerList(msg)
 		cmd.Println(jsonrpc.MarshalJSON(res))
+	} else if detailed == -2 {
+		showPrettyAndShort(cmd, msg)
 	} else {
 		res := jsonrpc.ConvShortPeerList(msg)
 		cmd.Println(jsonrpc.MarshalJSON(res))
 	}
 }
 
+func showPrettyAndShort(cmd *cobra.Command, msg *types.PeerList) {
+	usr, _ := user.Current()
+	dir := usr.HomeDir
+	aliasPath := filepath.Join(dir, AergoDir, NodeAliasFile)
+	// If the file doesn't exist, create it, or append to the file
```

Comment:
Does it really create the file?
The code does not append to the file, it only reads from it

Reply:
Yes, it does not write file. The comment seems confusing. I meant that the user, who excutes `aergocli`, should create the aliasNames file and put to the right place - `~/.aergo/` by himself.

Reply:
I will correct the comment.